icu4c/source/i18n/vtzone.cpp

```diff
@@ -1065,6 +1077,9 @@ VTimeZone::operator!=(const TimeZone& that) const {
 VTimeZone*
 VTimeZone::createVTimeZoneByID(const UnicodeString& ID) {
     VTimeZone *vtz = new VTimeZone();
+    if (vtz == nullptr) {
```

Comment:
The file is somewhat inconsistent in terms of using `NULL` or `nullptr`.
I tried to use the existing convention for each function in order to minimize churn.
However, I could just blanket change to use `nullptr` if that would be more clear.

Reply:
I personally think nullptr is a case where moving forward is better than consistency.

Reply:
Thanks, I'll change to use `nullptr` through out the entire file while I'm at it.